torch/distributed/elastic/rendezvous/c10d_rendezvous_backend.py

```diff
@@ -143,7 +143,7 @@ def _create_tcp_store(params: RendezvousParameters) -> TCPStore:
     else:
         is_host = _matches_machine_hostname(host)
 
-    use_libuv = params.get_as_bool("use_libuv", False)
+    use_libuv = params.get_as_bool("use_libuv", True)
```

Comment:
cc @kurman this line enables libuv backend by default in torch elastic

Reply:
Do you think we test this in Phabricator early on since we have E2E tests there?